Settings.cs

```diff
@@ -431,6 +431,69 @@ public static Settings Instance
         public float TwinsProbability { get; set; } = 0.03f;
+        [XmlElement]
+        [SettingProperty("Enable Character Fertility Probability Tweak", "Allows for adjusting for the probability to get pregnant, this will apply to everyone. By enabling this, we can use Min/Max Pregnancy Age")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Fertility Probability Tweak", true)]
+        public bool CharacterFertilityProbabilityTweakEnabled { get; set; } = false;
+        [XmlElement]
+        [SettingProperty("Character Fertility Probability", 0f, 1.0f, "Native value is 0.95. Determines the chance of getting pregnant")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Fertility Probability Tweak")]
+        public float CharacterFertilityProbability { get; set; } = 0.95f;
+        [XmlElement]
+        [SettingProperty("Enable Daily Chance Pregnancy Tweak", "Enabling this will completely override the daily pregnancy check. All settings below will be applied!")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak", true)]
+        public bool DailyChancePregnancyTweakEnabled { get; set; } = false;
+        [XmlElement]
+        [SettingProperty("Enable Player Character Fertility", "Is the player character (you) fertile? Native: true")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak")]
+        public bool PlayerCharacterFertileEnabled { get; set; } = true;
+        [XmlElement]
+        [SettingProperty("Min Pregnancy Age", 0, 999, "Minimum Age the Hero can get pregnant. Native: 18")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak")]
+        public int MinPregnancyAge { get; set; } = 18;
+        [XmlElement]
+        [SettingProperty("Max Pregnancy Age", 0, 999, "Maximum Age the Hero can get pregnant. Native: 45")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak")]
+        public int MaxPregnancyAge { get; set; } = 45;
+        [XmlElement]
+        [SettingProperty("Enable Max Children Tweak", "Allows to set the maximum number of children that you can get")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak")]
+        public bool MaxChildrenTweakEnabled { get; set; } = false;
+        [XmlElement]
+        [SettingProperty("Max Children", 0, 999, "Maximum number of children anyone can have. Default: 5")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak")]
+        public int MaxChildren { get; set; } = 5;
+        #endregion
+
+        #region Age Patches Tweak
+        [XmlElement]
+        [SettingProperty("Enable Age Tweaks", "Enables the tweaking of character age behaviour")]
+        [SettingPropertyGroup("Age Tweaks", true)]
+        public bool DefaultAgeTweakEnabled { get; set; } = true;
+        [XmlElement]
+        [SettingProperty("Become Infant Age", 0, 125, "Native: 3")]
+        [SettingPropertyGroup("Age Tweaks")]

```

Comment:
Use the subgroups as in the Pregnancy Settings:
[SettingPropertyGroup("Age Tweaks/Become Infant Age")]

Reply:
I am confused why I should do that?
Was there a change on the modlib code?

Reply:
I haven't checked how the new subgrouping works exactly. I think a main group for all age related changes should be fine, if it looks good :)

Reply:
Ignore this, I see where you are going with this now.